EmailHandler.php

```diff
@@ -95,22 +95,30 @@ public function validate(array $payload)
             return false;
         }
 
-        // The user must exists in Kanboard
-        $user = $this->userModel->getByEmail($payload['sender']);
-
-        if (empty($user)) {
-            $this->logger->info('Mailgun: ignored => user not found');
-            return false;
-        }
-
         // The project must have a short name
+        //    DMM: Moved project check in front of user check - to support project unknown user mappings
         $project = $this->projectModel->getByEmail($payload['recipient']);
 
         if (empty($project)) {
             $this->logger->info('Mailgun: ignored => project not found');
             return false;
         }
 
+        // The user must exists in Kanboard
+        $user = $this->userModel->getByEmail($payload['sender']);
+
+	// Check to see if a catchall user was specified - if the original sender is unrecognized
+	if (empty($user)) {
```

Comment:
There is some indentation issue here. Make sure it's consistent with the rest of the code.

Reply:
fixed